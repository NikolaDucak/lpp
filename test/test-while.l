//while

// entering
var a = true;
while( a ){
	println("enter");
	a = false;
}

// non entering
while( a ){
	println("shouldn't be printed");
}

// boolifying num 
// num != 0 is true
a = 2;
while( a ){
	println(a);
	a = 0;
}

// boolifying string
var b = "not empty";
while( b ){
	println(b);
	b = "";
}

