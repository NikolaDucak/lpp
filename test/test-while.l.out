enter
2
not empty
